sklearn/datasets/_california_housing.py

```diff
@@ -78,15 +79,22 @@ def fetch_california_housing(data_home=None, download_if_missing=True,
 
         .. versionadded:: 0.20
 
+    as_frame : boolean, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+
```

Comment:
Pleased add the directive:

```
.. versionadded:: 0.23
```

for after the paragraph explaining `as_frame`.